src/inverseRobot.js

```diff
@@ -25,6 +25,21 @@
  */
 function inverseRobot(robot) {
   // write code here
+  try {
+    return Object.fromEntries(
+      Object.entries(robot).reduce((newRobot, [value, key]) => {
+        if (newRobot.filter(x => x[0] === key).length) {
+          return null;
+        } else {
+          return newRobot.concat([
+            [key, value],
+          ]);
+        }
+      }, [])
+    );
```

Comment:
This solution may work, but it's too complicated.
Rewrite your solution using `for ... in` loop and create object using simple `{}`.